applications/trilinos_application/python_scripts/trilinos_linear_solver_factory.py

```diff
@@ -1,57 +1,39 @@
 from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
-from KratosMultiphysics import *
-from KratosMultiphysics.TrilinosApplication import *
 
-#
+def _CheckIfTypeIsDeprecated(config):
+    '''function to translate old/deprecated names to new names
+    needed for backwards-compatibility
+    '''
+    solver_type = config["solver_type"].GetString()
+
+    old_new_name_map = {
+        "CGSolver" : "cg",
+        "BICGSTABSolver" : "bicgstab",
+        "GMRESSolver" : "gmres",
+        "AztecSolver" : "aztec",
+        "MLSolver" : "multi_level",
+        "MultiLevelSolver" : "multi_level",
+        "AmgclMPISolver" : "amgcl",
```

Comment:
@RiccardoRossi is it ok for you to register the mpi-version of amgcl with the same name as the serial one?
This way you don't have to change it when switching to mpi
This is already done e.g. for gmres or bicgstab

=> which silver you get is still unique bcs in mpi the trilinos-lin-solver-factory is asked for the solver, and that one does not have the serial verison

Reply:
The name is the same?, sorry

Reply:
ping @RiccardoRossi

Reply:
Ping

Reply:
The ping is for @RiccardoRossi or for me?

Reply:
@RiccardoRossi

Reply:
@RiccardoRossi I plan to merge this tmr bcs it is blocking more developments that should go to the release

The question for you was only abt naming (I am unifying so I don't think you would object) but we could also discuss it afterwards if you want sth else